Library/Homebrew/extend/os/mac/diagnostic.rb

```diff
@@ -425,6 +425,20 @@ def check_if_supported_sdk_available
           #{update_instructions}
         EOS
       end
+
+      def check_for_bad_python_symlink
```

Comment:
I don't think we need this on macOS either given we'll need to handle the Linux behaviour anyway.

Reply:
Okay, well that makes this easier for sure.